README.md

```diff
@@ -1,2 +1,40 @@
-# nlp-in-10-lines
-Repository for PyCons 2016 workshop Natural Language Processing in 10 Lines of Code.
+### PyCon UK 2016 
+![NLP in 10 Lines of Code](http://i.imgur.com/TuusTAm.png)
+# Natural Language Processing in 10 Lines of Code
+##### from Andraz Hribernik, Matic Horvat, Aeneas Wiener, Tom Sherborne
```

Comment:
In my mind this was mostly an SEO thing to help your name come up when people search for spaCy but I can also take this out 😄